### ISSUE TITLE
Guest and Host QM calculations chemically incorrect.

### ISSUE THREAD

**Issue by lvotapka** [anandojha/qmmmrebind#5]
At this time, the code writes host/guest QM regions that are placed in the "host_qm.com" that are chemically incorrect. For instance, these images are from the structures passed into the Gaussian program:
![problem](https://user-images.githubusercontent.com/16157869/138175806-d8d08c95-a57d-451d-99f5-17a1de117af2.png)
Notice the circled groups. The terminal carboxylate has only one atom of oxygen, and the terminal amino group on the other end has only one hydrogen. Clearly these structures are not chemically possible and are, without a doubt, badly affecting the Gaussian calculations. A hydrogen should be added to the amine (maybe two), and an OH (or just an O) should be added to the carboxylate.

**Comment by anandojha** [comment]
Thanks, Lane. Yeah, this happens because the residues are not terminal, making bonds with the other residues. The code selects the atoms of the residues but not of the other. The user must either add the terminal atoms manually through the gaussview interface or simply use amber programs (addH) to add hydrogens and then optimize. This issue did not arise for host-guest systems since all the residues were selected. I will be writing a function to address this issue very soon, but for now, I am manually adding hydrogens to the system before gaussian optimization.

**Comment by anandojha** [comment]
Yeah, and thank you for pointing that out. I will be leaving this issue as unresolved when I add the function for the same. I remember describing this scenario to you even before we started this project.
